api/src/main/java/org/pmiops/workbench/iam/IamService.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.iam;
+
+public interface IamService {
```

Comment:
I remember we discussed this previously, but I don't remember where we landed: there is no remove API here. For granting the service account actAs - this isn't a problem. However, isn't it an issue to leave them with workflow runner role?

If it's not a problem, can you please add a writeup somewhere, e.g. to a doc or ticket about why we're not removing the role after unshare?

Reply:
Created https://precisionmedicineinitiative.atlassian.net/browse/RW-7615 to track this. Comments are in `WorkspaceController#shareWorkspace`
But it might not be a launch blocker if 
(1) Only User pet SA can actAs pet SA. 
(2) After unshare, user is not able to access the pet SA in workbench.

Reply:
Thanks. It will be difficult for them to use these roles via the Workbench product, but it will not be impossible for them to use it directly. I would err on the side of caution with a role like this and remove, but doesn't need to be in this PR